### ISSUE TITLE
Add alert levels

### ISSUE THREAD

**Issue by metbril** [gieljnssns/buienalarm-sensor-homeassistant#17]
The domoticz plugin provides alert levels

https://github.com/Xorfor/Domoticz-Buienalarm-Plugin

Add support for sensors for level and text to prevent users having to create template sensors themselves.

**Comment by nelbs** [comment]
that should be a nice PR 😄

**Comment by gieljnssns** [comment]
Zou inderdaad mooi zijn, maar echt bitter weinig tijd voor...

**Comment by StevenLooman** [comment]
Kan dit niet met een [Automation](https://www.home-assistant.io/docs/automation/basics/)?

**Comment by ShadowJonathan** [comment]
@StevenLooman no, the sensor doesn't expose values in the future, and it only stops at the next point that *any* rain falls.
